pytorch_lightning/trainer/predict_loop.py

```diff
@@ -24,6 +27,28 @@ def __init__(self, trainer):
         self.max_batches = None
         self.num_dataloaders = None
         self.warning_cache = WarningCache()
+        self.batch_indices: Optional[List[int]] = None
+        self.epoch_batch_indices: Optional[List[List[int]]] = None
+        self._return_predictions = not trainer.training_type_plugin.use_spawn
+        self._previous_grad_status: Optional[bool] = None
+
+    @property
+    def return_predictions(self) -> bool:
+        return self._return_predictions
+
+    @return_predictions.setter
+    def return_predictions(self, return_predictions: Optional[bool] = None) -> None:
+        if return_predictions and self.trainer.training_type_plugin.use_spawn:
+            raise MisconfigurationException(
+                "`return_predictions` should be set to `False` when using spawn accelerators. "
+                f"Found {return_predictions}"
+            )
+        if return_predictions is not None:
+            self._return_predictions = return_predictions
+
+    @property
+    def should_store_predictions(self) -> bool:
```

Comment:
why extra property with direct mapping to another property?

Reply:
It was from the PR 3/3. Removed !